intel/runtimes/tensorflow/ubi9-python-3.9/Dockerfile

```diff
@@ -0,0 +1,64 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+LABEL name="odh-notebook-intel-runtime-pytorch-ubi9-python-3.9" \
+    summary="Runtime Intel® optimized pytorch notebook image for ODH notebooks" \
```

Comment:
shouldn't that say "...optimized Tensorflow notebook....?

Reply:
Fixed. Thank you.